wandb/__init__.py

```diff
@@ -17,7 +17,8 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-__version__ = "0.12.11.dev1"
+# __version__ = "0.12.11.dev1"
+__version__ = "0.12.8"
```

Comment:
We'll want to get rid of this change before merging.

Reply:
oh yeah.. i was just testing the warning message and accidentally committed it! thanks for catching this